src/ServicePulse.Host/app/js/directives/ui.particular.multi-checkboxlist.js

```diff
@@ -10,23 +10,23 @@
             Deselection: 2
         });
 
-        $scope.selectRow = function(row, value){
+        function selectRow(row, value){
```

Comment:
I would actually prefer these as local vars instead of functions so that they don't get hoisted 😜